supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,53 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+.Technical best practices
+
+* Use hyperlinks consistently.
+* Do not use non hyperlinks.
+* Do not use JavaScript links. 
+* Do not use Flash links.
+* Do not use links in Frames or iframes.
+* Default to followable, crawlable links. Do not use “nofollow” links unless absolutely necessary.
+* Include links only when necessary.
+
+.Link text best practices
+
+* Contextually describe what the user will find at the target location so that they can decide if they want to leave their current location. 
+* Avoid irrelevant link text.
+* You can use a sentence or sentence fragment as the link text.
+
+
+
 [[internal-links]]
 == Internal links
 
+Internal links are links within www.redhat.com.
+
+You can use one of the following example link structures for consistency.
+
+.Example AsciiDoc for internal links
+----
```

Comment:
I think we're missing an intro sentence here, like "To link to another guide or to a user story within the same guide:"

Reply:
Done, thanks.

Reply:
Worth mentioning the escape character if one wants to hardcode a URL. Not sure if `www.redhat.com` needs to be hardcoded here.